dm/dm/ctl/master/operate_validation.go

```diff
@@ -0,0 +1,109 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package master
+
+import (
+	"context"
+	"errors"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/pingcap/tiflow/dm/dm/ctl/common"
+	"github.com/pingcap/tiflow/dm/dm/pb"
+)
+
+func NewIgnoreValidationErrorCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "ignore-error <task-name> <error-id|--all>",
```

Comment:
how about only allow keyword args for new added cli? such as  `dmctl validation ignore-error --task-name=test --error-id=1 --all=true`

we can discuss in this [rfc](https://github.com/pingcap/tiflow/pull/4458#discussion_r805488938)

this will make it much easier to write code for check args

Reply:
It sounds more intuitive and plausible to make user distinguish these two args. But the original one looks more succinct. @D3Hunter PTAL

Reply:
those 2 parameters are both mandatory, flags are better for optional ones, i prefer the original one.

Reply:
Reasonable